recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
   script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
can we switch to pip here

Reply:
Most likely, I wasn't going to because I wanted to make this PR minimally invasive. If a CI passes I'll put in pip.

Reply:
This means we need `{{ compiler("c") }}` as well, right?

Reply:
I think it is already in there.